.circleci/config.yml

```diff
@@ -129,6 +129,31 @@ jobs:
       - store_test_results:
           path: target/reports/
 
+  itest-s3-stream-provider:
```

Comment:
still missing the run config, but for now we run the whole test suite with the stream provider so it does not matter, I'll push once the tests passed